app/serializers/state_poverty_data_serializer.rb

```diff
@@ -0,0 +1,55 @@
+class StatePovertyDataSerializer < ActiveModel::Serializer
+  attributes :id,
+             :year,
+             :population,
+             :population_below_poverty,
+             :male_below_poverty,
+             :male_below_poverty_under5,
+             :male_below_poverty_5,
+             :male_below_poverty_6to11,
+             :male_below_poverty_12to14,
+             :male_below_poverty_15,
+             :male_below_poverty_16to17,
+             :male_below_poverty_18to24,
+             :male_below_poverty_25to34,
+             :male_below_poverty_35to44,
+             :male_below_poverty_45to54,
+             :male_below_poverty_55to64,
+             :male_below_poverty_65to74,
+             :male_below_poverty_over75,
+             :female_below_poverty,
+             :female_below_poverty_under5,
+             :female_below_poverty_5,
+             :female_below_poverty_6to11,
+             :female_below_poverty_12to14,
+             :female_below_poverty_15,
+             :female_below_poverty_16to17,
+             :female_below_poverty_18to24,
+             :female_below_poverty_25to34,
+             :female_below_poverty_35to44,
+             :female_below_poverty_45to54,
+             :female_below_poverty_55to64,
+             :female_below_poverty_65to74,
+             :female_below_poverty_over75,
+             :state,
+             :percent_below_poverty,
+             :percent_male_below_poverty,
+             :percent_female_below_poverty
+
```

Comment:
only added three percentages... wasn't sure if we needed all of them... someone else can add them if they really want more percentages :)